custom_components/opensprinkler/__init__.py

```diff
@@ -341,7 +340,6 @@ async def async_will_remove_from_hass(self):
         await super().async_will_remove_from_hass()
         self._coordinator.deregister_time_interval_listener()
 
-    @Throttle(SCAN_INTERVAL)
```

Comment:
@Throttle didn't cause issues in my testing and I use it in my custom components, what is the reason for removal?

Reply:
Throttle from my understanding is to stop it going too fast, for example limiting updates that call API too fast, so if it is < 30 secs it doesn't do anything

Reply:
changed it slightly